packages/grafana-ui/src/components/Logs/LogLabels.tsx

```diff
@@ -23,7 +23,7 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       padding: 0 2px;
       background-color: ${theme.colors.bg2};
       border-radius: ${theme.border.radius};
-      margin: 1px 4px 0 0;
+      margin: 1.75px 4px 0 0;
```

Comment:
I am wondering if we need to change styling here, cause Log Labels is used at multiple places and it is going to miss-align this at other places. Can't we fix it directly in the cell?

Reply:
It is for example used here 
<img width="374" alt="image" src="https://user-images.githubusercontent.com/30407135/174287142-5b1ad0e3-70ab-4f9f-8b16-5784a3c42eb9.png">

Reply:
Good catch! I moved it to select the first `span` in the labels-cell instead. Do you think this is better?

Reply:
Yeah! I think it make sense here.